keylime/ima.py

```diff
@@ -82,6 +82,12 @@ def _validate_ima_sig(exclude_regex, ima_keyring, allowlist, digest: ima_ast.Dig
     return valid_signature
 
 
+def _validate_ima_buf(exclude_regex, allowlist, digest: ima_ast.Digest, path: ima_ast.Name, data: ima_ast.Buffer):
+    # disregard data for now
+    del data
```

Comment:
This is not needed IMHO. Is OK to ignore it in the call.

Reply:
pylint requires it...

Reply:
Aha ... I think that there are maybe better options, like renaming to `_data` or disabling the pylint complain.

Reply:
`_data` would also work.